lib/main.dart

```diff
@@ -17,14 +13,7 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Stocket',
       debugShowCheckedModeBanner: false,
-      initialRoute: '/wallet',
-      routes: {
-        '/wallet': (context) => const Wallet(),
-        '/stockPurchased': (context) => const StockPurchased(),
-        '/stockMarket': (context) => const StockMarket(),
-        '/creditCard': (context) => const CreditCard(),
-        '/accountBalance': (context) => const AccountBalance(),
```

Comment:
Mais comment fait-on pour passer d'un screen à l'autre si les routes ne sont pas indiquées ?

Reply:
For the record : les navigations se fait à l'appel des constructeurs sur les boutons par exemple.
Point d'attention : il faudra éviter les appels excessifs de l'API lors des changements de screen.